content/api/inline-seeds.apib

```diff
@@ -0,0 +1,511 @@
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Either match or exclude has to be specified"
+                }
+            ]
+        }
+        ```
+
+### Update Seed Config [PUT /inline-seeds/config]
+
+Make updates to config details for the authenticated account.
+Only one seed config with 5 match rules max is currently supported.
+Auth context (from API key) provides customer and subaccount ids.
+
++ Request - Remove redundant rule (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "my-*"
+          ]
+        }
+        ```
+
++ Response 200
+    + Body
+
+        ```json
+        {
+          "results": {
+            "threshold": 1000,
+            "duration": 20,
+            "reset": 24,
+            "match": [
+              "my-*"
+            ],
+            "exclude": []
+          }
+        }
+        ```
+
++ Request - Too many matches (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "fred-*",
+            "wilma-*",
+            "pebbles-*",
+            "barney-*",
+            "betty-*",
+            "bam-bam-*"
+          ]
+        }
+        ```
+
++ Response 422 (application/json)
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Match rule limit exceeded (found 6, max is 5)"
+                }
+            ]
+        }
+        ```
+
++ Request - Both match and exclude (application/json)
+
+    + Headers
+
+        Authorization: 14ac5499cfdd2bb2859e4476d2e5b1d2bad079bf
+
+    + Body
+
+        ```json
+        {
+          "threshold": 1000,
+          "duration": 20,
+          "reset": 24,
+          "match": [
+            "fred-*",
+            "wilma-*"
+          ],
+          "exclude": [
+            "barney-*",
+            "betty-*"
+          ]
+        }
+        ```
+
++ Response 422 (application/json)
+
+    + Body
+
+        ```json
+        {
+            "errors": [
+                {
+                    "message": "Incompatible rule type combination: 'match' and 'exclude' are mutually exclusive."
+                }
+            ]
+        }
+        ```
+
+### Check Campaign Eligibility [GET /inline-seeds/match?id={campaign_id}]
+
+Check `campaign_id` against config to determine seed eligibility.
+Auth context (from API key) provides customer and subaccount ids.
+If `PUT /inline-seeds/options` was used to disable this feature all campaigns are considered ineligible.
+
++ Parameters
+    + id: `my-campaign` (string, required)
+
++ Request
```

Comment:
In the sample request, is it possible to use a sample campaign ID rather than `{campaign_id}`?